apps/arweave/src/ar_peers.erl

```diff
@@ -149,15 +152,49 @@
 start_link() ->
 	gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
 
+%%--------------------------------------------------------------------
 %% @doc Return the list of peers in the given ranking order.
-%% Rating is an estimate of the peer's effective throughput in bytes per millisecond.
+%%      Rating is an estimate of the peer's effective throughput in
+%%      bytes per millisecond.
 %%
-%% 'lifetime' considers all data ever received from this peer and is most useful when we care
-%% more about identifying "good samaritans" rather than maximizing throughput (e.g. when
-%% polling for new blocks are determing which peer's blocks to validated first).
+%%      `lifetime' considers all data ever received from this peer and
+%%      is most useful when we care more about identifying "good
+%%      samaritans" rather than maximizing throughput (e.g. when
+%%      polling for new blocks are determing which peer's blocks to
+%%      validated first).
 %%
-%% 'current' weights recently received data higher than old data and is most useful when we care
-%% more about maximizing throughput (e.g. when syncing chunks).
+%%      `current' weights recently received data higher than old data
+%%      and is most useful when we care more about maximizing throughput
+%%      (e.g. when syncing chunks).
+%%
+%%      `connected' returns connected `lifetime' peers.
+%%
+%%      `{timestamp, Seconds}' filters `current' list based on their
+%%      last connection, where `Seconds' is the number of seconds
+%%      since the last connection.
+%%
+%%      `{current, Days}' filters `current' list based on
+%%      their last connection where `Days' is a number of days.
```

Comment:
Since for now we only have 2 lists that we care about `lifetime` and `current` I suggest we only implement support for those two lists when querying `get_peers`. As we identify more rankings that are necesary, we can extend the list of supported rankings - but I think for now we only have the 2?